### PR TITLE
Quests page

### PR DESCRIPTION
**Pull request by clintonhill**
Quest page ajax done.

### CODE REVIEW

File: public/stylesheets/navbar.css [hunk 1]
```diff
@@ -22,11 +22,12 @@ nav {
...
 
 .logo-container {
-    display: inline-block;
+    /* display: inline-block; */
     width: 200px;
-    height: 50px;
-    background-repeat: no-repeat;
-    background-size: cover;
+    height: 80px;
+    /* background-repeat: no-repeat; */
```
Comment (Lazytangent) [inline public/stylesheets/navbar.css]: Could you remove this comment?

File: public/stylesheets/navbar.css [hunk 2]
```diff
@@ -22,11 +22,12 @@ nav {
 }
 
 .logo-container {
-    display: inline-block;
+    /* display: inline-block; */
```
Comment (Lazytangent) [inline public/stylesheets/navbar.css]: Could you remove this comment?

File: public/javascripts/xp.js [hunk 3]
```diff
@@ -30,15 +30,28 @@ function updateXPBar() {
 document.addEventListener('DOMContentLoaded', () => {
   updateXPBar();
 
-  const submitBtn = document.querySelector('#_4');
-
-  submitBtn.addEventListener('click', async event => {
-    const res = await fetch('/api/quests/4', {
-      method: 'PUT',
+  const completeButtons = Array.from(document.querySelectorAll('.complete-quest-button'));
+  //console.log(completeButtons)
```
Comment (Lazytangent) [inline public/javascripts/xp.js]: Could you remove this please?

**Review by Lazytangent — CHANGES_REQUESTED** [review]
Looks good, please remove the comments

File: public/stylesheets/navbar.css (hunk 1 above)
Comment (clintonhill) [reply public/stylesheets/navbar.css]: Done.

File: public/stylesheets/navbar.css (hunk 2 above)
Comment (clintonhill) [reply public/stylesheets/navbar.css]: Done

File: public/javascripts/xp.js (hunk 3 above)
Comment (clintonhill) [reply public/javascripts/xp.js]: Done

**Comment by clintonhill** [comment]
Changes made.

**Review by Lazytangent — APPROVED** [review]
Thank you very much!

**Pull request merged**